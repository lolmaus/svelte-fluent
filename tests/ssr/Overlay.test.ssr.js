import Component from '../../stories/overlay/Variables.svelte'

it('renders correctly', () => {
  const result = Component.render({ productName: 'example product' })
  expect(result).toMatchSnapshot()
})
