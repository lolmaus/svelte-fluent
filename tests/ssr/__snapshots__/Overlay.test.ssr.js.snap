// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Object {
  "css": Object {
    "code": "",
    "map": null,
  },
  "head": "",
  "html": "<span>You can download ⁨example product⁩ by clicking
on the <strong>Download</strong> button or read
the <a data-l10n-name=\\"release-notes\\" href=\\"https://example.com/\\" target=\\"_blank\\">release notes</a> to learn more.</span>",
}
`;
